cds_migrator_kit/videos/weblecture_migration/transform/models/video_lecture.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Document Server.
+# Copyright (C) 2024 CERN.
+#
+# Invenio is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License as
+# published by the Free Software Foundation; either version 2 of the
+# License, or (at your option) any later version.
+#
+# Invenio is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""CDS-Videos Video Lecture model."""
+
+from cds_migrator_kit.rdm.migration.transform.models.overdo import CdsOverdo
+
+from .base import model as base_model
+
+
+class VideoLecture(CdsOverdo):
+    """Translation Index for CERN Video Lectures."""
+
+    __query__ = '8567_.x:"Absolute master path" 8567_.d:/mnt/master_share* -980__.C:MIGRATED -980__.c:DELETED -5831_.a:digitized'
+
+    __ignore_keys__ = {
+        "110__a",  # corporate author
```

Comment:
could you order the ignore keys lines? it would be easier to read (in each of your subcategories, not all of them)